runtime/src/main/java/org/corfudb/runtime/Backup.java

```diff
@@ -43,7 +45,7 @@
     private final String backupTempDirPath;
 
     // The stream IDs of tables which are backed up
-    private final List<UUID> streamIDs;
+    private List<UUID> streamIDs;
```

Comment:
why not final?

Reply:
Done. I made some temporary changes and forgot to bring it back.